Site/ASAB Maestro/Descriptors/kibana.yaml

```diff
@@ -99,7 +99,7 @@ nginx:
     - proxy_set_body ""
     - proxy_set_header X-Request-URI "request_uri"
     - proxy_cache kibana
-    - proxy_pass http://upstream-seacat-auth-public/batman/nginx?client_id=kibana
```

Comment:
This is upstream part of nginx configuration from maestro. There is no upstream-seacat-auth-private. But it might be helpful and you could create one in the upstreams section of nginx configuration in seacat-auth descriptor
```
root@lmc01:~# cat /opt/site/nginx-1/conf.d/upstreams.conf 
# GENERATED FILE!

upstream upstream-seacat-auth-public {
	keepalive 32;
	server lmc01:3081;
}

upstream upstream-seacat-auth-8900 {
	keepalive 32;
	server lmc01:8900;
}

upstream upstream-influxdb {
	keepalive 32;
	server lmc01:8086;
}

upstream upstream-asab-config-8894 {
	keepalive 32;
	server lmc01:8894;
	server lmc02:8894;
	server lmc03:8894;
}

upstream upstream-asab-governator-8892 {
	keepalive 32;
	server lmc01:8892;
	server lmc02:8892;
	server lmc03:8892;
}

upstream upstream-asab-library-8893 {
	keepalive 32;
	server lmc01:8893;
	server lmc02:8893;
	server lmc03:8893;
}

upstream upstream-asab-remote-control-8891 {
	keepalive 32;
	server lmc01:8891;
	server lmc02:8891;
	server lmc03:8891;
}
```

Reply:
i added `upstream-seacat-auth-private` in the descriptor, but i guess it is the same thing as the `api` section (both use port 8900). i don't know how exactly the descriptor is interpreted so please correct me if this is not right